package.json

```diff
@@ -13,14 +13,21 @@
   },
   "scripts": {
     "build": "babel src --out-dir dist",
+    "lint": "eslint src/**/*.js",
```

Comment:
It would be better to put `eslint .` as the script and have a `.eslintignore` with:
```
build
test/app/**
```
Because your specs should also be tested by `eslint`. Always better to exclude with a file than later change this script to include more stuff